cunumeric/module.py

```diff
@@ -1176,6 +1176,239 @@ def squeeze(a: ndarray, axis: Optional[NdShapeLike] = None) -> ndarray:
     return a.squeeze(axis=axis)
 
 
+def broadcast_shapes(*args: Sequence(Union[tuple(int), int])) -> tuple(int):
+    """
+
+    Broadcast the input shapes into a single shape.
+
+    Parameters
+    ----------
+    `*args` : tuples of ints, or ints
+        The shapes to be broadcast against each other.
+
+    Returns
+    -------
+    tuple : Broadcasted shape.
+
+    See Also
+    --------
+    numpy.broadcast_shapes
+
+    Availability
+    --------
+    Multiple GPUs, Multiple CPUs
+
+    """
+    return np.broadcast_shapes(*args)
+
```

Comment:
do we plan to implement our own broadcast_shapes method?

Reply:
I think this is not necessary because cuNumeric will call `np.broadcast_shapes` if there's no implementation for the routine. I'll delete this.

Reply:
No need to delete this, I think, because someone might want to call cunumeric.broadcast_shapes. My question was: do we plan to reuse broadcast_shapes from numpy or implement our own?

Reply:
if you look at the numpy implementation of the `broadcast_shapes`, it calls to `np.empty` and `np.broadcast` etc. I thought we might want to replace this with cunumeric implementation of the method

Reply:
@magnatelee : what do you think?

Reply:
> if you look at the numpy implementation of the `broadcast_shapes`, it calls to `np.empty` and `np.broadcast` etc. I thought we might want to replace this with cunumeric implementation of the method

Yeah, I've looked at the numpy impl and it seems not optimal for `broadcast_shapes`.
I think we can keep this skeleton function calling just the numpy implementation and make another small PR to have a new implementation.

Reply:
If you decide to do this, please create a JIRA task for broadcast_shapes